### DISCUSSION TITLE
Add current installed version of the custom dashboard

### DISCUSSION THREAD

**Discussion by macsatcom** [jheredianet/Teslamate-CustomGrafanaDashboards#69] [Ideas]
It would be nice to have an easy way of telling if I currently have the newest version installed, or if it is time to make a new pull from git.

**Comment by jheredianet** [comment]
Hi, this is a good idea. I think "Database Information" dashboard is a good candidate to have that information. 
Keep in touch, I'll implement it in the new release.

**Reply by stauffenberg2020** [reply]
@jheredianet , But this dashboard has been moved out of this repo right?

**Reply by jheredianet** [reply]
Yes, it has. I’v it’s now as a core dashboard of Teslamate. 
What I did Is to release versions with the date of publishing to recognize its version

**Reply by stauffenberg2020** [reply]
But how can a user figure out which version has been installed in their setup?

**Reply by jheredianet** [reply]
> But how can a user figure out which version has been installed in their setup?

It's on the README.md page. You can run this command at your shell console:

```bash
cat ~/Teslamate-CustomGrafanaDashboards/README.md | grep "Last release:"
```
(replace ~/Teslamate-CustomGrafanaDashboards with your own path where you installed the dasboards).

You should see something like this:

![image](https://github.com/user-attachments/assets/91d20110-108a-41d5-8901-f05c7a629cc4)

**Reply by stauffenberg2020** [reply]
Are there any plans to add it to any of the dashboards? so that the info is available straight to Grafana just like any other car and/or database stats?

**Reply by jheredianet** [reply]
> Are there any plans to add it to any of the dashboards? so that the info is available straight to Grafana just like any other car and/or database stats?

I'll trying to figure it out, using the cloud, so I have to change it once and not in every dashboard.
